test/xmltest/__snapshots__/not-wf.test.js.snap

```diff
@@ -0,0 +1,1407 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`xmltest/not-wellformed standalone should match 001.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc>
+&lt;doc
+?
+&lt;a
+</doc>",
+  "error": Array [
+    "[xmldom error]	element parse error: Error: invalid attribute:?
+@#[line:2,col:1]",
+    "[xmldom error]	element parse error: Error: invalid tagName:a<
+@#[line:4,col:1]",
+  ],
+  "warning": Array [
+    "[xmldom warning]	attribute \\"?\\" missed value!! \\"?\\" instead2!!
+@#[line:2,col:1]",
+  ],
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 002.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc>
+&lt;.doc>
+</doc>",
+  "error": Array [
+    "[xmldom error]	element parse error: Error: invalid tagName:.doc
+@#[line:2,col:1]",
+  ],
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 003.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc><? ?></doc>",
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 004.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc>&lt;?target some data></doc>",
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 005.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc>&lt;?target some data?</doc>",
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 006.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc><!-- a comment -- another --></doc>",
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 007.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc>&amp;amp no refc</doc>",
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 008.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc>&amp;.entity;</doc>",
+}
+`;
+
+exports[`xmltest/not-wellformed standalone should match 009.xml with snapshot 1`] = `
+Object {
+  "actual": "<doc>{!0!}</doc>",
```

Comment:
The issue is that putting them like this might again render them as "invisible" depending on what engine displays them. I explicitly rendered them to "ask for attention".